cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/batch/distributed/MapReduceContainerLauncher.java

```diff
@@ -94,8 +94,7 @@ public void run() {
             try {
               ((AutoCloseable) classLoader).close();
             } catch (Exception e) {
-              System.err.println("Failed to close ClassLoader " + classLoader);
-              e.printStackTrace();
+              LOG.error("Failed to close ClassLoader {}", classLoader, e);
```

Comment:
should it be `"Failed to close ClassLoader {} with exception : {}"`

Reply:
Done